### PR TITLE
Review url

### PR DESCRIPTION
**Pull request by daisieh**
When an old review URL is used, you now get a message pointing at the archived item’s DOI.

### CODE REVIEW

File: dspace/modules/xmlui/src/main/java/org/dspace/app/xmlui/aspect/submission/DryadReviewTransformer.java [hunk 1]
```diff
@@ -152,62 +156,67 @@ public void addBody(Body body) throws SAXException, WingException, UIException,
...
-            div.addPara().addButton("submit_simple_item_info").setValue(T_showsimple);
-
-            div.addHidden("submit_full_item_info").setValue("true");
-        }
 
+            DCValue[] vals = wfItem.getItem().getMetadata("dc.title");
+            ReferenceSet referenceSet = null;
+
+            if (showfull == null) {
+                referenceSet = div.addReferenceSet("narf", ReferenceSet.TYPE_SUMMARY_VIEW);
```
Comment (ryscher) [inline dspace/modules/xmlui/src/main/java/org/dspace/app/xmlui/aspect/submission/DryadReviewTransformer.java:185]: ?

**Comment by ryscher** [comment]
Submitted a new item to review workspace on my VM. 
Received DOI 10.5072/FK2dryad.dk30b. 
Opened a new browser (not logged in). 
Resolved URL http://localhost:9999/review?doi=10.5072/FK2dryad.dk30b
Incorrectly received the message "This item has been archived at...", even though the item was still in review.

**Comment by ryscher** [comment]
This functionality does work correctly when using the correct form of the URL
http://localhost:9999/review?doi=doi:10.5072/FK2dryad.dk30b

**Comment by ryscher** [comment]
This is ok to merge, but we should still fix the issue above when the URL does not include "doi:". I'll leave it up to @daisieh whether to correct the issue before merging or correct it in a separate PR.

**Comment by daisieh** [comment]
I mean, wouldn't we have to go around issuing incorrect review URLs for that to be a problem? I'm thinking this is fine to merge if it works with URLs of the form we're worried about having escaped into the wild.

**Comment by elizabethdryad** [comment]
I agree with Ryan that the "doi:" issue should be corrected at some point.
Some journals are aware of how to construct review urls, and leaving out
the "doi:" part would be a likely error for them to make. If they did that
and got a message that the item had been archived...mass confusion. Thanks!

On Thu, Oct 29, 2015 at 1:05 AM, Daisie Huang notifications@github.com
wrote:

> Merged #1031 https://github.com/datadryad/dryad-repo/pull/1031.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/datadryad/dryad-repo/pull/1031#event-448843864.

## 

_Elizabeth Hull_
Operations Manager, Dryad
ehull@datadryad.org
http://datadryad.org/ https://datadryad.org/

**Pull request merged**